presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueMetastoreModule.java

```diff
@@ -50,10 +50,10 @@ protected void setup(Binder binder)
     {
         configBinder(binder).bindConfig(GlueHiveMetastoreConfig.class);
 
-        OptionalBinder.newOptionalBinder(binder, GlueColumnStatisticsProvider.class)
+        newOptionalBinder(binder, GlueColumnStatisticsProvider.class)
                 .setDefault().to(DisabledGlueColumnStatisticsProvider.class).in(Scopes.SINGLETON);
 
-        OptionalBinder.newOptionalBinder(binder, Key.get(RequestHandler2.class, ForGlueHiveMetastore.class));
+        newOptionalBinder(binder, Key.get(RequestHandler2.class, ForGlueHiveMetastore.class));
```

Comment:
Do this cleanup commit first, so that we don't add new code and then immediately modify it